.github/workflows/dotnet.yml

```
@@ -0,0 +1,43 @@
+name: Test backend
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)